### ISSUE TITLE
Tests timing out 

### ISSUE THREAD

**Issue by jywarren** [publiclab/Leaflet.DistortableImage#681]
Seeing this come up a lot; and @sashadev-sky tried working on it in https://github.com/publiclab/Leaflet.DistortableImage/pull/609#issuecomment-613778918

I attempted a fix in https://github.com/publiclab/Leaflet.DistortableImage/pull/675 

https://docs.travis-ci.com/user/gui-and-headless-browsers/#karma-and-firefox-inactivity-timeouts

```
    browserNoActivityTimeout: 40000,
```

But we're still seeing it in https://github.com/publiclab/Leaflet.DistortableImage/pull/680 (you can try it out in fact in GitPod there!)

The error is typically:

```
30 06 2020 19:24:15.577:WARN [PhantomJS 2.1.1 (Linux x86_64)]: Disconnected (1 times), because no message in 30000 ms.
PhantomJS 2.1.1 (Linux x86_64) ERROR
  Disconnected, because no message in 30000 ms.
```

And after #675 it is now:

```
  Disconnected, because no message in 40000 ms.
```

Any help appreciated!

**Comment by MLefebvreICO** [comment]
Hi there,

Just found your issue while searching for why our tests suite on our end was not working anymore.

First of all, let me suggest you use Chrome headless instead of PhantomJS which was abandonned years ago.

But for now even Chrome headless is broken, because of their latest stable release ..

You can follow on this here : https://bugs.chromium.org/p/chromium/issues/detail?id=1090988

**Comment by jywarren** [comment]
Ah, thank you for the tip! Looking in our test setup, i'm thinking we could make the switch here perhaps:

https://github.com/publiclab/Leaflet.DistortableImage/blob/33f0bf58b7176fefbf7bda99552686b862028331/test/karma.conf.js#L98-L111

**Comment by jywarren** [comment]
Looks like there are `karma-chrome-launcher` and `karma-firefox-launcher` among others!

**Comment by jywarren** [comment]
So i would guess the steps might be:

1. include `karma-chrome-launcher` in `package.json`
2. add "Chrome" to `browsers:` array
3. change this include:

https://github.com/publiclab/Leaflet.DistortableImage/blob/33f0bf58b7176fefbf7bda99552686b862028331/test/karma.conf.js#L15

**Comment by jywarren** [comment]
Let's see how this does, and if Chrome bug resolves we can update it: https://github.com/publiclab/Leaflet.DistortableImage/pull/685

**Comment by sashadev-sky** [comment]
closed via #716
